test/parser.spec.ts

```diff
@@ -114,33 +114,18 @@ describe('gfm parser', () => {
 
     const expected = [
       notion.paragraph([notion.richText('hello')]),
-      notion.paragraph([
-        notion.richText('public class Foo {}', {
-          annotations: {code: true},
-        }),
-      ]),
+      notion.code([notion.richText('public class Foo {}')]),
     ];
 
     expect(actual).toStrictEqual(expected);
   });
 
   it('should parse block quote', () => {
-    const ast = md.root(
-      md.blockquote(
-        md.heading(1, md.text('hello '), md.emphasis(md.text('world')))
-      )
-    );
+    const ast = md.root(md.blockquote(md.paragraph(md.text('hello'))));
 
     const actual = parseBlocks(ast);
 
-    const expected = [
-      notion.headingOne([
-        notion.richText('hello '),
-        notion.richText('world', {
-          annotations: {italic: true},
-        }),
-      ]),
```

Comment:
Can this part of this test be added back? Or is it tested elsewhere?

Reply:
Same as above - it is just the adjustment of the test to support the fact that block quotes can only contain parahraphs with inline elements.

Reply:
Following up from above, I would want the expected to look something like 
```
const expected = [
      notion.blockquote([
        notion.richText('hello '),
        notion.richText('world', {
          annotations: {italic: true},
        }),
      ]),
```

Reply:
Ok - let me take a look!